### ISSUE TITLE
Wordpress 5.5 - wpColorPickerL10n is not defined

### ISSUE THREAD

**Issue by levonmanucharian** [CMB2/CMB2#1368]
After updating to Wordpress 5.5, getting the following javascript error in admin panel page edit section: 
Uncaught ReferenceError: wpColorPickerL10n is not defined. The location is: cmb2/js/wp-color-picker-alpha.js line 24.

Not browser-related, tested with 3-4 major browsers. Please advice.

**Comment by tw2113** [comment]
Can you provide your CMB2 fields configuration?

I just grabbed the example version of the ColorPicker from https://github.com/CMB2/CMB2/wiki/Field-Types#colorpicker and was able to use it without issue on WP 5.5.

**Comment by levonmanucharian** [comment]
> Can you provide your CMB2 fields configuration?
> 
> I just grabbed the example version of the ColorPicker from https://github.com/CMB2/CMB2/wiki/Field-Types#colorpicker and was able to use it without issue on WP 5.5.

I don't use any specific configuration, here's an example of a color field:

array(
    'name' => esc_html__('Background Color', 'eventorian'),
    'id' => $this->prefix .'product_bgcolor',
    'type' => 'colorpicker',
    'default' => '#f9f9f9',
),

The plugin is incorporated into the theme, if that helps. Everything was working fine before the WP 5.5 update.

**Comment by tw2113** [comment]
I wonder if the copy bundled with the theme is on an older version than what i have and the latest release. So the theme developer would need to update as well.

**Comment by levonmanucharian** [comment]
> I wonder if the copy bundled with the theme is on an older version than what i have and the latest release. So the theme developer would need to update as well.

No, it's the latest version of CMB2 from Github master, also checked the dev branch to make sure nothing is missing.

**Comment by tw2113** [comment]
Hmm. I now wonder if there's some other part of the site in question that is causing conflicts somehow. Is this a site that at all possible could have other plugins disabled and whatnot? Get it temporarily reduced down as much as possible to see if the issue goes away?

**Comment by levonmanucharian** [comment]
> Hmm. I now wonder if there's some other part of the site in question that is causing conflicts somehow. Is this a site that at all possible could have other plugins disabled and whatnot? Get it temporarily reduced down as much as possible to see if the issue goes away?

Thanks, I will dig deeper and provide an update if I find anything.

**Comment by rejuancse** [comment]
Hello, In your theme functions.php file add this code,
`if( is_admin() ){
	add_action( 'wp_default_scripts', 'wpmm_custom_scripts' );
	function wpmm_custom_scripts( $scripts ){
		$scripts->add( 'wp-color-picker', "/wp-admin/js/color-picker-suffix.js", array( 'iris' ), false, 1 );
		did_action( 'init' ) && $scripts->localize(
			'wp-color-picker',
			'wpColorPickerL10n',
			array(
				'clear'            => __( 'Clear' ),
				'clearAriaLabel'   => __( 'Clear color' ),
				'defaultString'    => __( 'Default' ),
				'defaultAriaLabel' => __( 'Select default color' ),
				'pick'             => __( 'Select Color' ),
				'defaultLabel'     => __( 'Color value' ),
			)
		);
	}
}`

I hope problem will be solved.

**Comment by levonmanucharian** [comment]
> Hello, In your theme functions.php file add this code,
> `if( is_admin() ){ add_action( 'wp_default_scripts', 'wpmm_custom_scripts' ); function wpmm_custom_scripts( $scripts ){ $scripts->add( 'wp-color-picker', "/wp-admin/js/color-picker-suffix.js", array( 'iris' ), false, 1 ); did_action( 'init' ) && $scripts->localize( 'wp-color-picker', 'wpColorPickerL10n', array( 'clear' => __( 'Clear' ), 'clearAriaLabel' => __( 'Clear color' ), 'defaultString' => __( 'Default' ), 'defaultAriaLabel' => __( 'Select default color' ), 'pick' => __( 'Select Color' ), 'defaultLabel' => __( 'Color value' ), ) ); } }`
> 
> I hope problem will be solved.

Rejuan, thanks so much for the help, that fixed the issue for me.

**Comment by tw2113** [comment]
5.5.1 likely helped resolve this temporarily, but like with #1373 we may still need to address some newest changes from WP core.
